app/assets/javascripts/modules/improve-this-page.js

```diff
@@ -0,0 +1,160 @@
+(function (Modules) {
+  "use strict";
+
+  function ImproveThisPage () {
+    this.controller = null;
+    this.view = null;
+
+    this.start = function ($element) {
+      this.view = new View($element);
+      this.controller = new Controller(this.view);
+
+      this.controller.init();
+    }
+  };
+
+  function View ($element) {
+    var that = this;
+
+    this.pageIsUsefulButton = $element.find('.js-page-is-useful');
```

Comment:
Do we prefix the names of jquery objects with `$` here, or did I imagine that? There's a number of places throughout this file where that would apply if so.